src/js/pages/ChapterReview/components/ReviewColumn/SelectedTake.js

```diff
@@ -6,24 +6,58 @@ export default class SelectedTake extends React.Component {
 
   constructor(props) {
     super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.isActive = this.isActive.bind(this);
   }
 
+  handleClick() {
+    const {updateActiveChunkIndex, activeChunkIndex,
+      index, resetTake, takesPlaying} = this.props;
+    updateActiveChunkIndex(activeChunkIndex, index, takesPlaying);
+    resetTake(true);
+  }
 
-  render() {
+  isActive() {
     const {take, updateActiveChunkIndex, resetPos,txt,
       activeChunkIndex, active, resetTake, selectedTakesLength} = this.props;
-    return (
-      <Container>
-        <Take take={take} txt={txt}
-          updateActiveChunkIndex={updateActiveChunkIndex}
-          activeChunkIndex={activeChunkIndex}
-          active={active} selectedTakesLength={selectedTakesLength}
-          resetPos={resetPos}
-          resetTake={resetTake} />
+    if (active) { //return the selected take without the onClick handler
+      return (
+        <Container>
+          <Take take={take} txt={txt}
+            updateActiveChunkIndex={updateActiveChunkIndex}
+            activeChunkIndex={activeChunkIndex}
+            active={active} selectedTakesLength={selectedTakesLength}
+            resetPos={resetPos}
+            resetTake={resetTake} />
+
+          <Comments active={active} txt={txt} comments= {take.publishedTake.comments} />
+
+        </Container>
+      );
+    }
+
+    else {
+      return (
+        <Container onClick={this.handleClick}>
+          <Take take={take} txt={txt}
+            updateActiveChunkIndex={updateActiveChunkIndex}
+            activeChunkIndex={activeChunkIndex}
+            active={active} selectedTakesLength={selectedTakesLength}
+            resetPos={resetPos}
+            resetTake={resetTake} />
 
-        <Comments active={active} txt={txt} comments= {take.publishedTake.comments} />
+          <Comments active={active} txt={txt} comments= {take.publishedTake.comments} />
 
-      </Container>
+        </Container>
+      );
+    }
+  }
+
+  render() {
+    let selectedTake = this.isActive();
```

Comment:
when the variable does not get re-assigned you could use "const" instead of "let".  not an issue but in the future you could implement it

Reply:
you're right. I just did that in the rush and didn't pay attention. Thank you